src/redist/targets/BundledManifests.targets

```diff
@@ -104,6 +104,21 @@
     
     <Copy SourceFiles="@(ManifestContent)"
           DestinationFolder="$(RedistLayoutPath)sdk-manifests/%(DestinationPath)"/>
-  
+
+    <ItemGroup>
+      <FormattedManifestPaths Include="$(RedistLayoutPath)sdk-manifests\**\%(BundledManifests.Identity)\WorkloadManifest.json" />
+    </ItemGroup>
+
+    <JsonPropertyParser JFileNames="@(FormattedManifestPaths)" JPaths="version" Value="empty">
+      <Output TaskParameter="Value" ItemName="BaselineManifestsWithVersion" />
+    </JsonPropertyParser>
+
+    <ItemGroup>
+      <FormattedBaselineManifest Include="{" />
+      <FormattedBaselineManifest Include="&quot;$([System.IO.Path]::GetFileName(%(BaselineManifestsWithVersion.Identity)))&quot;: &quot;%(BaselineManifestsWithVersion.Version)&quot;,"/>
+      <FormattedBaselineManifest Include="}" />
+    </ItemGroup>
+
+    <WriteLinestoFile File="$(RedistLayoutPath)sdk-manifests\BaselineManifest.json" Lines="@(FormattedBaselineManifest)" />
```

Comment:
I think this needs to be named something like baseline.workloadset.json and needs to include a version as it is a set of all the baseline manifests.

Take a look at https://github.com/dotnet/designs/pull/294/files#diff-ab104db694a9386180073fed5271729b627ada894f9f78cffbd0020ff15425a1R44

So it needs to be laid out in a versioned folder. @dsplaisted is that correct?